airflow-core/src/airflow/models/deadline.py

```diff
@@ -100,35 +102,6 @@ def add_deadline(cls, deadline: Deadline, session: Session = NEW_SESSION):
         session.add(deadline)
 
 
-class DeadlineReference(Enum):
```

Comment:
Do we need to leave an import in this module for compatibility?

Reply:
I don't think so.  This was added it in the previous PR purely as a placeholder so I could give an example of how something else will be used and the note on L107 made that pretty clear.  Nobody should have been using it for anything, I think it's safe to just drop it?